pyais/messages.py

```diff
@@ -961,10 +964,10 @@ class MessageType22Broadcast(Payload):
     # If the message is broadcast (addressed field is 0),
     # the ne_lon, ne_lat, sw_lon, and sw_lat fields are the
     # corners of a rectangular jurisdiction area over which control parameter
-    ne_lon = bit_field(18, int, from_converter=from_course, to_converter=to_course, default=0, signed=True)
-    ne_lat = bit_field(17, int, from_converter=from_course, to_converter=to_course, default=0, signed=True)
-    sw_lon = bit_field(18, int, from_converter=from_course, to_converter=to_course, default=0, signed=True)
-    sw_lat = bit_field(17, int, from_converter=from_course, to_converter=to_course, default=0, signed=True)
+    ne_lon = bit_field(18, float, from_converter=from_course, to_converter=to_course, default=0, signed=True)
```

Comment:
are these lat/long as well, and should use suitable to/from functions?

Reply:
ne_lon, ne_lat, sw_lon, and sw_lat fields are in 0.1 minutes. [Source](https://gpsd.gitlab.io/gpsd/AIVDM.html#_type_22_channel_management)